runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/HotKeyLoggerTest.java

```diff
@@ -50,11 +50,11 @@ public void SetUp() {
   }
 
   @Test
-  public void correctHotKeyMessage() {
+  public void correctHotKeyMessageWithoutKey() {
     HotKeyLogger hotKeyLogger = new HotKeyLogger(clock);
 
     assertFalse(hotKeyLogger.isThrottled());
-    String m = hotKeyLogger.getHotKeyMessage("TEST_STEP_ID", "1s");
+    String m = hotKeyLogger.getHotKeyMessage("TEST_STEP_ID", "1s", null);
```

Comment:
You can use ExpectedLogs as a test rule to capture whether output was produced or not instead of abstracting out a getHotKeyMessage. See https://github.com/apache/beam/blob/cfca7f176acb3542c7b86312f968819a1e97866e/sdks/java/core/src/test/java/org/apache/beam/sdk/coders/SerializableCoderTest.java#L259 for an example.

Reply:
Done